code-studio/src/js/pd/workshop_dashboard/components/workshop_table.jsx

```diff
@@ -0,0 +1,109 @@
+/* global React */
+
+var WorkshopTableRow = require('./workshop_table_row.jsx');
```

Comment:
You are right that we haven't been entirely consistent (I'm the guilty party that created those components). Not that big a deal if these are different if changing will be a lot of work.